src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/ActivityHelper.cs

```diff
@@ -43,14 +43,16 @@ internal static class ActivityHelper
         /// Try to get the started <see cref="Activity"/> for the running <see
         /// cref="HttpContext"/>.
         /// </summary>
-        /// <param name="httpContext"><see cref="HttpContext"/>.</param>
+        /// <param name="context"><see cref="HttpContext"/>.</param>
         /// <param name="aspNetActivity">Started <see cref="Activity"/> or <see
         /// langword="null"/> if 1) start has not been called or 2) start was
         /// called but sampling decided not to create an instance.</param>
         /// <returns><see langword="true"/> if start has been called.</returns>
-        public static bool HasStarted(HttpContext httpContext, out Activity aspNetActivity)
+        public static bool HasStarted(HttpContext context, out Activity aspNetActivity)
         {
-            object itemValue = httpContext.Items[ActivityKey];
+            Debug.Assert(context != null, "Context should be non-null.");
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.debug.assert?view=net-5.0#System_Diagnostics_Debug_Assert_System_Boolean_System_String_

It seems the .NET practice is to put "what actually happened" instead of "what is expected" in the second argument.

Reply:
Updated